content/it/docs/concepts/containers/container-environment.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Container Environment
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Questa pagina descrive le risorse disponibili nei Container eseguiti in Kubernetes.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Container environment
+
+Quando si esegue un Container in Kubernetes, le seguenti risorse sono rese disponibili:
+
+* Un filesystem, composto dal file system dell'[image](/docs/concepts/containers/images/) e da uno o più [volumes](/docs/concepts/storage/volumes/).
+* Una serie di informazioni sul Container stesso.
+* Una serie di informazioni sugli oggetti nel cluster.
+
+### Informazioni sul Container
+
+L' *hostname* di un Container è il nome del Pod all'interno del quale è eseguito il Container.
+È consultabile tramite il comando `hostname` o tramite la funzione
+[`gethostname`](http://man7.org/linux/man-pages/man2/gethostname.2.html)
+disponibile in libc.
+
+Il nome del Pod e il namespace possono essere resi disponibili come environment variables attraverso l'uso
```

Comment:
Do Italian readers prefer “variabile d'ambiente” or “environment variable”?

Reply:
I'm for using English for technical terms.